conda_libmamba_solver/utils.py

```diff
@@ -55,3 +56,18 @@ def is_channel_available(channel_url) -> bool:
     except Exception as exc:
         log.debug("Failed to check if channel %s is available", channel_url, exc_info=exc)
         return False
+
+
+def compatible_matchspecs(*match_specs) -> bool:
+    """
+    Returns True if all match_specs are compatible with each other, False otherwise.
+    """
+    if len(match_specs) < 2:
+        raise ValueError("Must provide at least two match_specs")
+    if len({ms.name for ms in match_specs if ms.name not in (None, "", "*")}) > 1:
+        return False
```

Comment:
Wait, that means if there are more than one ms with the same name, or different names, this just fails?

Reply:
If you pass `python=3.9` and `scipy=1.10`, they are not compatible because they are different packages. That's what I am trying to do. But I am debugging the failing test and this is not doing what I want later in the function (e.g. `python=3.8` and `python=3.9` are incompatible pin-wise, but this returns `python 3.8.*,3.9.*`, which is nooot what I want 😂).